CONTRIBUTING.md

```diff
@@ -102,6 +102,14 @@ We generally allow backports of following changes only to the latest supported b
 - Improvements to CI signal
 - Improvements to the test framework
 
+While we recommend to target following type of changes to the next minor release, CAPI maintainers will also consider
```

Comment:
IMO we're not going to come to a wording here that can really define what kinds of changes are "small" or  "limited and well-scoped". 

I liked the idea of needing X number of maintainers - I think @CecileRobertMichon mentioned 2 on the community meeting - so we have some guardrails on what counts as limited in scope. That gives a clear responsibility for who interprets "limited" etc.

Another idea is to have these backports communicated in the community meeting which will raise visibility of how often they're happening - similar to features which impact providers being announced in the CAPI community meeting.

Reply:
@CecileRobertMichon @killianmuldoon I have rephrased stating that lgtm from two maintainers is required, PTAL